hikari/__init__.py

```diff
@@ -30,6 +30,8 @@
 
 import os as _os
 
+from hikari import api
+from hikari import impl
```

Comment:
What does this achieve?

Reply:
That behaviour is specific to MYPY, not all type checkers will see api or impl as members on the imported module `hikari` unless you explicitly export them, same applies to every other module and the current behaviour is that you have to directly import them if you're using something like pyright which as far as I could tell was what you'd intended anyways since well you just don't explicitly export modules within other modules at all. This is just a special case since the contents of those modules is mostly left not exposed on the top level so actually being able to access them from the hikari import should be common enough